packages/react/src/component-tests/IcToggleButtonGroup/IcToggleButtonGroup.cy.tsx

```diff
@@ -0,0 +1,209 @@
+/* eslint-disable react/jsx-no-bind */
+/// <reference types="Cypress" />
+
+import React from "react";
+// import { IcToggleButtonGroup, IcToggleButton } from "../../components";
+import { CYPRESS_AXE_OPTIONS } from "../../../cypress/utils/a11y";
+import { mount } from "cypress/react";
+import * as test from "./IcToggleButtonGroupTestData";
+import {
+  HAVE_ATTR,
+  NOT_HAVE_ATTR,
+  NOT_HAVE_BEEN_CALLED,
+} from "../utils/constants";
+
+const DEFAULT_TEST_THRESHOLD = 1;
+const IC_TOGGLE_BUTTON_GROUP = "ic-toggle-button-group";
+const IC_TOGGLE_BUTTON = "ic-toggle-button";
+
+const getToggle = (number: number) => {
+  return cy.get(IC_TOGGLE_BUTTON_GROUP).find(IC_TOGGLE_BUTTON).eq(number);
+};
+
+/**
+ * Justification for additional rule: in the browser this is
+ * not reported as an issue by axe Devtools, but it upsets cypress
+ */
+const TOGGLE_BUTTON_AXE_OPTIONS = {
+  rules: {
+    ...CYPRESS_AXE_OPTIONS.rules,
+    "nested-interactive": { enabled: false },
+  },
+};
+
+const WIN_CONSOLE_SPY = "@spyWinConsoleLog";
+
+describe("IcToggleButtonGroup", () => {
+  describe("E2E", () => {
+    it("Single/manual should check only one toggle", () => {
+      mount(<test.ToggleGroupSingle />);
```

Comment:
I think it makes sense to just import each of these in separately from `.IcToggleButtonGroupTestData` so that all of the tests are consistent
```suggestion
      mount(<ToggleGroupSingle />);
```